taskapp/src/components/login/login.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import { Redirect } from 'react-router-dom';
+import auth from '../../auth';
+
+class LoginComponent extends Component {
+	constructor(props) {
+		super(props);
+		this.username = '';
```

Comment:
There is no need for these properties, there are better ways to handle input fields.